server.js

```diff
@@ -2,29 +2,103 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+// Using my own data set that shows the dogs of Zurich, Switzerland
+
+import dogs from './data/dogs-in-zurich.json'
+
+// Explaining the different values
+// ALTER -> Age
+// GESCHLECHT -> Gender
+// STADTKREIS -> City Quarter or District
+// RASSE1 -> Dog's Primary Breed
+// RASSE2 -> Dog's Secondary Breed
+// GEBURTSJAHR_HUND -> Dog's Year of Birth
+// GESCHLECHT_HUND -> Dog's Gender
+// HUNDEFARBE -> Dog's Color
+
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// My routes
+// "First page" - the "about" route - giving instructions on how to interpret the json for non german speaking.
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send("<strong>Hello!</strong> This project shows all living dogs in Zurich, Switzerland. <br><br> <i>Explaning the different values.</i> <ul><li>ALTER -> Age.</li> <li>GESCHLECHT -> Gender.</li> <li>STADTKREIS -> City Quarter or District.</li> <li>RASSE1 -> Dog's Primary Breed.</li> <li>RASSE2 -> Dog's Secondary Breed.</li> <li>GEBURTSJAHR_HUND -> Dog's Year of Birth.</li> <li>GESCHLECHT_HUND -> Dog's Gender.</li> <li>HUNDEFARBE -> Color of the Dog.</li></ul> <h4>Instructions</h4> Enter '/dogs' to see all dogs. Enter '/dogs/{id}' to single out ONE dog. Ex. '/dogs/126'. Enter '/year/{year}' to see all dogs born that year. Ex. '/year/2011'. <br> Enter '/breed/{breed}' to see all dogs of a certain breed. Ex. '/breed/Weimaraner' ")
```

Comment:
You can create the index.html file and render it, or use templating engine to generate nice view:D 
You can read more about it here: https://ejs.co/